<ui version="4.0" >
 <class>ManagerWidget</class>
 <widget class="QWidget" name="ManagerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>597</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="projectButton" >
       <property name="text" >
        <string>Project Management</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="songButton" >
       <property name="text" >
        <string>Song Management</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="audioSourcesButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Audio Sources</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="globalPropertiesButton" >
       <property name="text" >
        <string>Settings</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>141</width>
         <height>431</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../traverso.qrc" />
 </resources>
 <connections/>
</ui>
